### PR TITLE
Update sample format guidelines. 

### PR DESCRIPTION
**Pull request by kurtisvg**

### CODE REVIEW

File: dlp/src/main/java/dlp/snippets/InspectImageFile.java [hunk 1]
```diff
@@ -34,11 +34,15 @@
 
 public class InspectImageFile {
 
+  public static void inspectImageFile() {
```
Comment (lesv) [inline dlp/src/main/java/dlp/snippets/InspectImageFile.java]: I like this.

File: dlp/src/main/java/dlp/snippets/InspectImageFile.java [hunk 2]
```diff
@@ -48,13 +52,9 @@ public static void inspectImageFile(String projectId, String filePath) {
...
       // Specify the type and content to be inspected.
       ByteString fileBytes = ByteString.readFrom(new FileInputStream(filePath));
-      ByteContentItem byteItem = ByteContentItem.newBuilder()
-          .setType(BytesType.IMAGE)
-          .setData(fileBytes)
-          .build();
-      ContentItem item = ContentItem.newBuilder()
-          .setByteItem(byteItem)
-          .build();
+      ByteContentItem byteItem =
```
Comment (lesv) [inline dlp/src/main/java/dlp/snippets/InspectImageFile.java]: I prefer the line by line over a single line (easier to understand).

Comment (kurtisvg) [reply dlp/src/main/java/dlp/snippets/InspectImageFile.java]: I agree - these changes were done unintentionally by `google-java-format`. I'll revert when I clean this up.

File: dlp/src/main/java/dlp/snippets/InspectImageFile.java (hunk 1 above)
Comment (nnegrey) [reply dlp/src/main/java/dlp/snippets/InspectImageFile.java]: +1

**Comment by kurtisvg** [comment]
Added examples for the new overloaded format requirement, and did some general cleanup + examples for some other things in `SAMPLE_FORMAT.md`.

Some of the DLP tests are still failing, but it's unrelated to these updates. 

@lesv - I clarified some info about `Exceptions` to try to clear up some bad practices we are seeing. I'm going to leave the exception changes we discussed during the last weekly sync up to you, as I'm still not sure the best way to clarify when to throw vs catch for these type of snippets.

File: SAMPLE_FORMAT.md [hunk 3]
```diff
@@ -44,44 +47,95 @@ Project should have a `pom.xml` that is readably formatted, declares a parent po
...
 Example:
 ```java
 try {
   // Do something
-} catch (IllegalArgumentException e) {
-  // IllegalArgumentException's are thrown when an invalid argument has been passed to a function.
-  // This error should be logged to that the root cause can be debugged and prevented in the future.
-  System.out.println("Error during functionName: \n" + e.toString());
+} catch (IllegalArgumentException ok) {
+  // IllegalArgumentException's are thrown when an invalid argument has been passed to a function. Ok to ignore.
```
Comment (lesv) [inline SAMPLE_FORMAT.md]: my changes

**Pull request merged**